docs/modules/ROOT/pages/how-to-guides/managing-environments/proc_creating_your_own_environment.adoc

```diff
@@ -1,31 +1,63 @@
 = Creating your own environment 
 
-Use {ProductName} to continuously build and test your application. Use your own cluster as your deployment environment. By using your own cluster outside Red Hat, you have more control over where your application is deployed, and you can ensure it meets your specific needs. If you bring your own cluster you can still access all the features and benefits of {ProductName}. 
+You can configure {ProductName} to deploy applications to your own cloud-hosted cluster, outside the default development environment. By using this option to bring your own cluster (BYOC), you can deploy your applications to an environment that meets your specific needs. And if you bring your own cluster, you can still access all the features and benefits of {ProductName}. 
 
 .Prerequisites
 
-* You must have access to credentials with full read and write access to the namespace that you provided to {ProductName}.   
-* When you upload a `kubeconfig` file, it must include a credential with full read and write access to your provided namespace. 
-* The supplied `kubeconfig` file must use token-based authentication to connect to your cluster. Other mechanisms, such as certificate-based authentication, are not yet supported. 
-* The context in your `kubeconfig` file must point to the cluster you want to use at the time you upload the file. 
-
-.Procedures
-
-To create your own environment and upload your own `kubeconfig` file, complete the following steps in the {ProductName} UI: 
-
-. Go to *Environments* > *Create Environment*. 
-. Complete all the required fields for *Create Environment*.  
+* You must have credentials for full read and write access to the namespace for your cluster.   
```

Comment:
Full?
Do we want to say "with"?
If not is there an incomplete read and write access?